fabric-convention-tags-v2/src/main/java/net/fabricmc/fabric/api/tag/convention/v2/ConventionalEntityTypeTags.java

```diff
@@ -36,6 +36,10 @@ private ConventionalEntityTypeTags() {
 
 	public static final TagKey<EntityType<?>> MINECARTS = register("minecarts");
 	public static final TagKey<EntityType<?>> BOATS = register("boats");
+
+	/**
+ 	 * Tag containing entity types that extend AnimalEntity
```

Comment:
If this is the case, then Tadpole, Bats, Squid, Glow Squid, and all the fish needs to be removed from the tag.